.github/workflows/e2e-fork.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - name: Run e2e Test
         env:
           # see images here https://github.com/cosmos/relayer/pkgs/container/relayer/versions
-          RLY_TAG: "v2.0.0-rc2"
+          RLY_TAG: "v2.0.0"
```

Comment:
Can we add a var for this? Is that possible?

Reply:
this file is the tests that run on forks only, is this the one you want to paramaterize? If not, the existing `workflow_call` (a workflow re-usable in other workflows) has the relayer tag as a paramter. See [here](https://github.com/cosmos/ibc-go/blob/6922700c19c2b78409dbb2b6157c59ae890f8a4e/.github/workflows/e2e-test-workflow-call.yml#L34)

This is configurable in the [manually triggerable workflows](https://github.com/cosmos/ibc-go/blob/6922700c19c2b78409dbb2b6157c59ae890f8a4e/.github/workflows/e2e-manual-simd.yaml#L44)


The task that runs on every pr is [this one](https://github.com/cosmos/ibc-go/blob/6922700c19c2b78409dbb2b6157c59ae890f8a4e/.github/workflows/e2e.yaml#L44)

In the end the env var `RLY_TAG` is what's used to configure it.

Reply:
I'd like a single var to update when bumping the default relayer tag

Reply:
as it currently is there isn't one place to specify the relayer version. Unfortunately github actions doesn't really support sharing variables across workflows like that (from within the workflow files).

We could move this to a secret which would be configured via the UI, or we could do something like [this](https://arinco.com.au/blog/github-actions-share-environment-variables-across-workflows/)

and just maintain a file that we use to add env vars to all the runs.

Reply:
Although even this might not work, as we need to manually list items to provide the drop down (in the manual tasks) and as far as I know the only way to provide this is with an explicit list.

Reply:
Ah no worries, it was a minor nit. Not worth the extra hassle